python/lsst/afw/table/base/baseContinued.py

```diff
@@ -315,5 +315,15 @@ def __getattr__(self, name):
         except AttributeError:
             return getattr(self.columns, name)
 
+    def __str__(self):
+        if self.isContiguous():
+            return str(self.asAstropy())
+        else:
+            fields = ' '.join(x.field.getName() for x in self.schema)
```

Comment:
Why does the Astropy implementation print out rows (so you get something that looks like a table) and ours does not?